controllers/release/adapter_test.go

```diff
@@ -262,7 +270,7 @@ var _ = Describe("Release adapter", Ordered, func() {
 			Expect(adapter.release.IsReleased()).To(BeFalse())
 		})
 
-		It("should do nothing if the managed processing has not completed", func() {
+		It("should do nothing if the final processing has not completed", func() {
```

Comment:
The managed stuff on the It before this needs to be updated

Reply:
Changed the above it